__mocks__/styleMock.js

```diff
@@ -1 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2018-2018 Red Hat, Inc.
```

Comment:
2020?

Reply:
I got it from https://github.com/eclipse/che-dashboard.
 I can fix it)

Reply:
I have done it

Reply:
Maybe for that subproject `2020-2020` makes more sense than `2018-2020`?
@akurinnoy @olexii4 WDYT?

Reply:
https://github.com/che-incubator/che-dashboard-next/pull/25/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3
```
  "name": "che-dashboard",
  "version": "5.0.0",
  "description": "Dashboard for Eclipse CHE",
...
```
 '2018-2020' will be better in this case